pkg/resolver/search.go

```diff
@@ -283,6 +260,15 @@ func formatLabels(labels map[string]interface{}) string {
 	return strings.Join(labelStrings, ",")
 }
 
+func formatArray(labels []interface{}) string {
+	keys := make([]string, 0)
+	for _, k := range labels {
+		keys = append(keys, fmt.Sprintf("%s", k))
+	}
```

Comment:
We know the size of the array, so we can allocate it. *We won't ever notice the performance difference here, it's just for best practice.
```go
keys := make([]string, len(labels))
	for i, k := range labels {
		keys[i] = fmt.Sprintf("%s", k) //<-- I don't think this will work for non-strings.
	}
```

Reply:
Wait, why the diff isn't showing what's currently on main?
https://github.com/stolostron/search-v2-api/blob/main/pkg/resolver/search.go#L270-L284